Language Server fails to start with multiple --add-exports in java.jdt.ls.vmargs

The Language Server fails to start when multiple ```---add-exports``` are specified in the ```java.jdt.ls.vmargs``` setting.

For example these VSCode settings (in ```.vscode/settings.json```) start the Language Server correctly:

```
{
  "java.jdt.ls.vmargs": "--add-exports java.xml/com.sun.org.apache.xerces.internal.impl.dtd=ALL-UNNAMED",
  "java.trace.server": "verbose"
}
```

These settings do not:

```
{
  "java.jdt.ls.vmargs": "--add-exports java.xml/com.sun.org.apache.xerces.internal.impl.dtd=ALL-UNNAMED --add-exports java.xml/com.sun.org.apache.xerces.internal.xni.parser=ALL-UNNAMED --add-exports java.xml/com.sun.org.apache.xerces.internal.xni.grammars=ALL-UNNAMED --add-exports java.xml/com.sun.org.apache.xerces.internal.util=ALL-UNNAMED --add-exports java.xml/com.sun.org.apache.xerces.internal.parsers=ALL-UNNAMED --add-exports java.xml/com.sun.org.apache.xerces.internal.jaxp=ALL-UNNAMED"
  "java.trace.server": "verbose"
}
````

I see this error in the Language Support for Java output pane:

```
Error: Could not find or load main class java.xml.com.sun.org.apache.xerces.internal.xni.parser=ALL-UNNAMED
Caused by: java.lang.ClassNotFoundException: java.xml.com.sun.org.apache.xerces.internal.xni.parser=ALL-UNNAMED
```

I don't see any errors reported in the Chrome dev tools console, or in the Language Server workspace logs.

##### Environment
- Operating System:  Windows 11 and WSL2 Debian 12
- JDK version:  21.0.1
- Visual Studio Code version:  1.88.1
- Java extension version:  v1.30.2024041608 (pre-release)

##### Steps To Reproduce
1. Add multiple ```--add-exports``` arguments to the ```java.jdt.ls.vmargs``` value in VSCode ```settings.json```.
2. Open a Java project workspace.

##### Current Result

The Language Server fails to start, and reports the above error to the Output pane.

##### Expected Result

The Language Server starts with all ```--add-exports``` supplied to the JVM.

--- a/README.md
+++ b/README.md
@@ -269,7 +269,7 @@ For information on getting started, refer to the [CONTRIBUTING instructions](CON
 Continuous Integration builds can be installed from [http://download.jboss.org/jbosstools/jdt.ls/staging/](http://download.jboss.org/jbosstools/jdt.ls/staging/?C=M;O=D). Download the most recent `java-<version>.vsix` file and install it by following the instructions [here](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix).
 Stable releases are archived under http://download.jboss.org/jbosstools/static/jdt.ls/stable/.
 
-Also, you can contribute your own VSCode extension to enhance the existing features by following the instructions [here](https://github.com/redhat-developer/vscode-java/wiki/Contribute-a-Java-Extension).
+Also, you can contribute your own VS Code extension to enhance the existing features by following the instructions [here](https://github.com/redhat-developer/vscode-java/wiki/Contribute-a-Java-Extension).
 
 Feedback
 ===============

--- a/src/javaServerStarter.ts
+++ b/src/javaServerStarter.ts
@@ -312,9 +312,7 @@ export function parseVMargs(params: any[], vmargsLine: string) {
 		arg = arg.replace(/(\\)?"/g, ($0, $1) => { return ($1 ? $0 : ''); });
 		// unescape all escaped double quotes
 		arg = arg.replace(/(\\)"/g, '"');
-		if (params.indexOf(arg) < 0) {
-			params.push(arg);
-		}
+		params.push(arg);
 	});
 }
 

--- a/test/standard-mode-suite/extension.test.ts
+++ b/test/standard-mode-suite/extension.test.ts
@@ -134,11 +134,11 @@ suite('Java Language Extension - Standard', () => {
 
 		java.parseVMargs(vmArgs, userArgs);
 
-		assert.equal(4, vmArgs.length);
+		assert.equal(5, vmArgs.length);
 		assert.equal('-noverify', vmArgs[0]);
 		assert.equal('foo', vmArgs[1]);
 		assert.equal('-Xmx512m', vmArgs[2]);
-		assert.equal('-Dfoo=something with blank', vmArgs[3]);
+		assert.equal('-Dfoo=something with blank', vmArgs[4]);
 	});
 
 	test('should parse VM arguments with spaces', () => {

